runhouse/servers/http/http_server.py

```diff
@@ -612,6 +620,38 @@ async def rename_object(request: Request, params: RenameObjectParams):
                 e, traceback.format_exc(), from_http_server=True
             )
 
+    @staticmethod
```

Comment:
is there any reason to try and use one endpoint and add an operation here? 

no harm in just creating many endpoints, right? then we can make it more strict as to which parameters come in